input/system/tembo/system.go

```diff
@@ -0,0 +1,165 @@
+	cpuUsage, err := getFloat64(query, "https://api.data-1.use1.tembo.io/"+config.TemboNamespace+"/metrics/query?query=", client, headers)
+	if err != nil {
+		logger.PrintError("Tembo/System: Encountered error when getting CPU info %v\n", err)
+		return
+	}
+
+	system.CPUStats = make(state.CPUStatisticMap)
+	system.CPUStats["all"] = state.CPUStatistic{
+		DiffedOnInput: true,
+		DiffedValues: &state.DiffedSystemCPUStats{
+			UserPercent: cpuUsage,
+		},
+	}
+
+	// Get total memory
+	query = "sum(max by(pod) (kube_pod_container_resource_requests{job=\"kube-state-metrics\", namespace=\"" + config.TemboNamespace + "\", resource=\"memory\"}))"
+	memoryTotalBytes, err := getUint64(query, "https://api.data-1.use1.tembo.io/"+config.TemboNamespace+"/metrics/query?query=", client, headers)
+	if err != nil {
+		logger.PrintError("Tembo/System: Encountered error when getting memory info %v\n", err)
+		return
+	}
+
+	// Get available memory
+	query = "sum(max by(pod) (kube_pod_container_resource_requests{job=\"kube-state-metrics\", namespace=\"" + config.TemboNamespace + "\", resource=\"memory\"})) - sum(container_memory_working_set_bytes{job=\"kubelet\", metrics_path=\"/metrics/cadvisor\", namespace=\"" + config.TemboNamespace + "\",container!=\"\", image!=\"\"})"
+	memoryAvailableBytes, err := getUint64(query, "https://api.data-1.use1.tembo.io/"+config.TemboNamespace+"/metrics/query?query=", client, headers)
+	if err != nil {
+		logger.PrintError("Tembo/System: Encountered error when getting memory info %v\n", err)
+		return
+	}
+
+	system.Memory.TotalBytes = memoryTotalBytes
+	system.Memory.AvailableBytes = memoryAvailableBytes
+	system.Memory.FreeBytes = memoryAvailableBytes
+
+	// Get disk capacity
+	//TODO(ianstanton) Check if volume claim names differ in cases like HA
+	query = "kubelet_volume_stats_capacity_bytes{namespace=\"" + config.TemboNamespace + "\", persistentvolumeclaim=~\"" + config.TemboNamespace + "-1" + "\"}"
+	diskCapacity, err := getUint64(query, "https://api.data-1.use1.tembo.io/"+config.TemboNamespace+"/metrics/query?query=", client, headers)
+	if err != nil {
+		logger.PrintError("Tembo/System: Encountered error when getting disk info %v\n", err)
+		return
+	}
+
+	// Get disk available
+	//TODO(ianstanton) Check if volume claim names differ in cases like HA
+	query = "kubelet_volume_stats_available_bytes{namespace=\"" + config.TemboNamespace + "\", persistentvolumeclaim=~\"" + config.TemboNamespace + "-1" + "\"}"
+	diskAvailable, err := getUint64(query, "https://api.data-1.use1.tembo.io/"+config.TemboNamespace+"/metrics/query?query=", client, headers)
+	if err != nil {
+		logger.PrintError("Tembo/System: Encountered error when getting disk info %v\n", err)
+		return
+	}
+
+	diskUsed := diskCapacity - diskAvailable
+	system.DiskPartitions = make(state.DiskPartitionMap)
+	system.DiskPartitions["/"] = state.DiskPartition{
+		DiskName:      "md0",
```

Comment:
```suggestion
		DiskName: "default",
```

And I'd drop the PartitionName since its not needed here.